src/main/java/it/gov/pagopa/pu/bff/controller/OrganizationController.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.pu.bff.controller;
+
+import it.gov.pagopa.pu.bff.service.organization.BrokerServiceImpl;
+import it.gov.pagopa.pu.p4pa_organization.model.generated.PersonalisationFe;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class OrganizationController {
```

Comment:
you should implement the interface automatically generated from openApi

Reply:
updated